go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/viper v1.2.1
 	github.com/streadway/quantile v0.0.0-20150917103942-b0c588724d25 // indirect
+	github.com/stretchr/testify v1.3.0
```

Comment:
Why do we add a new dependency?

Reply:
It provides very handy helper functions for tests, like assert.Equal, e.g.: https://github.com/census-instrumentation/opencensus-service/pull/557/files#diff-ef311f1bc51e5169d51cfb73bc922052R181

Reply:
Please file an issue on the opentelemetry-go and service to use this then :) If it is that nice :)

Reply:
Yes, I think it is pretty useful and makes tests cleaner and shorter :-)

Do you want me to open issues for these 2 repos? 
https://github.com/open-telemetry/opentelemetry-go
https://github.com/open-telemetry/opentelemetry-service

Reply:
Yes, just to make sure we use this from the beginning in the new project

Reply:
Done:
https://github.com/open-telemetry/opentelemetry-go/issues/3
https://github.com/open-telemetry/opentelemetry-service/issues/7